ui-tests/tests/vwc-card/index.js

```diff
@@ -97,7 +97,7 @@ export async function createElementVariations(wrapper) {
 			<div slot="content">I'm the card content that overrides the content from attributes. I have no text styles or any styles for that metter :) You can do whatever you want here</div>
 		</vwc-card>
 
-		<vwc-card >
+		<vwc-card dp="24">
```

Comment:
Elevation u mean?

Reply:
Its a test for the dp change and for the content slot

Reply:
the _card_ api member should be `elevation`. in opposed to _elevation_ where its member is, indeed, `dp`

Reply:
ohhhhhh. Got it.
Fixed it